### PR TITLE
Removed unused strings

### PR DESCRIPTION
**Pull request by ghost**
Removed the following strings as per issue #9957

--Character.json--
displayPhoto
bodySlim
bodyBroad
unlockSet
bodyHead
bodySkin
bodyHair
hairBangs
ponytail
hairBase
basicSkins
equipmentBonusText
classBonusText
classEquipBonus
battleGearText
useCostumeInfo1
useCostumeInfo2
costumePopoverText
autoEquipPopoverText
gainedLevel
fullyHealed
levelBonusText
allocatedPointsText
buffsText
wizardText
optOutOfPMs
youDied
dieText
sureReset
displayNameDescription1
displayNameDescription2
displayNameDescription3
unequipBattleGear
unequipCostume
unequipPetMountBackground
chooseClassHeading
warriorWiki
mageWiki
rogueWiki
healerWiki
showQuickAllocation
hideQuickAllocation
quickAllocationLevelPopover

--CommunityGuidelines.Json--
iAcceptCommunityGuidelines

--Contents.json--
premiumPotionAddlNotes

--Contrib.json--
championSixth
championFifth
legSeventh
guardianText
heroicText
modalContribAchievement
helpedText1
helpedText2
visitHeroes
surveysSingle
surveysMultiple
currentSurvey
surveyWhen
blurbInbox
blurbGuildsPage
blurbChallenges

--DefaultTasks.json--
defaultHabit5Text
defaultHabit5Notes
defaultDaily1Text
defaultTodo2Text
defaultTodo2Notes
defaultReward2Text
defaultReward2Notes

--Challenge.json--
challengeWinner
deleteOrSelect
challengeDiscription
noNone
notParticipating
challengeTag
challengeTagPop
prizePop
prizePopTavern
selectGroup
sureDelCha
sureDelChaTavern
removeTasks
challengedOwnedFilterHeader
challengedOwnedFilter
challengedEitherOwnedFilter
backToChallenges
challengeNotEnoughGems
noPermissionEditChallenge
noPermissionDeleteChallenge
noPermissionCloseChallenge
noChallengeOwner
noChallengeOwnerPopover
noCompletedTodosChallenge
userTasksNoChallengeId
shortNameTooShort

--Generic.json--
titleTasks
titleAvatar
titleBackgrounds
titleStats
titleAchievs
titleProfile
titleInbox
titleTavern
titleParty
titleHeroes
titlePatrons
titleGuilds
titleChallenges
titleDrops
titleQuests
titlePets
titleMounts
titleEquipment
titleSettings
expandToolbar
collapseToolbar
markdownBlurb
showFormattingHelp
hideFormattingHelp
youType
youSee
emojiExample
markdownLinkEx
markdownImageEx
unorderedListHTML
unorderedListMarkdown
groupPlansTitle
newGroupTitle
moreInfoChallengesURL
moreInfoTagsURL
showMoreMore
showMoreLess
gemsWhatFor
notPorted
buyThis
noReachServer
errorUpCase
serverUnreach
seeConsole
endTour
audioTheme_danielTheBard
audioTheme_wattsTheme
audioTheme_gokulTheme
audioTheme_luneFoxTheme
audioTheme_rosstavoTheme
audioTheme_dewinTheme
audioTheme_airuTheme
audioTheme_beatscribeNesTheme
audioTheme_arashiTheme
audioTheme_maflTheme
audioTheme_pizildenTheme
checkOutProgress
greetingCardExplanation
greeting0
greeting1
greeting2
greeting3
greetingCardAchievementTitle
greetingCardAchievementText
enableDesktopNotifications
artDesign
booksWriting
comicsHobbies
diyCrafts
education
foodCooking
healthFitness
scienceTech

--Front.json--
accept1Terms
accept2Terms
alexandraQuote
althaireQuote
andeeliaoQuote
autumnesquirrelQuote
businessSample1
businessSample2
businessSample3
businessSample4
businessSample5
businessText
choreSample1
choreSample2
choreSample3
choreSample4
choreSample5
communityBug
communityFeature
communityKickstarter
companyPrivacy
companyTerms
companyVideos
contribUse
dragonsilverQuote
dreimQuote
elmiQuote
evagantzQuote
examplesHeading
featureAchievementByline
featureAchievementHeading
featureEquipByline
featureEquipHeading
featurePetByline
featurePetHeading
featureSocialByline
featureSocialHeading
featuresHeading
footerDevs
footerCommunity
footerCompany
frabjabulousQuote
gamifyButton
goalSample1
goalSample2
goalSample3
goalSample4
goalSample5
healthSample1
healthSample2
healthSample3
healthSample4
healthSample5
infhQuote
irishfeet123Quote
kazuiQuote
landingend
landingend2
landingp1
landingp2
landingp3
landingp4
landingp2header
landingp3header
landingp4header
loginAndReg
loginFacebookAlt
loginGoogleAlt
passMan
punishByline
punishHeading1
punishHeading2
questByline1
questByline2
questHeading1
questHeading2
rewardByline1
rewardByline2
rewardHeading
sampleDailies
sampleHabits
sampleToDo
schoolSample1
schoolSample2
schoolSample3
schoolSample4
schoolSample5
sixteenBitFilQuote
skysailorQuote
socialTitle
supermouse35Quote
teamSample1
teamSample2
teamSample3
testimonialHeading
localStorageClearing
localStorageClearingExplanation
unlockByline1
unlockByline2
unlockHeadline
watchVideos
zelahQuote
imagine1
landingCopy1
landingCopy2
landingCopy3
getStartedNow
altAttrNavLogo
altAttrLifehacker
altAttrNewYorkTimes
altAttrMakeUseOf
altAttrForbes
altAttrCnet
altAttrFastCompany
altAttrKickstarter
altAttrDiscover
altAttrFrabjabulous
altAttrAlexandraSo
altAttrEvaGantz
altAttrSupermouse35
altAttrAlthaire
altAttrInfH
altAttrDreiM
altAttrKazui
altAttrAutumnesquirrel
altAttrIrishfeet123
altAttrElmi
altAttr16bitFil
altAttrZelahMeyer
altAttrSkysailor
altAttrIonic
altAttrWebstorm
altAttrGithub
altAttrTrello
altAttrSlack

-Messages.json--
messageDuplicateTaskID
messageEquipped
messageAlreadyPurchasedGear
messageAuthPasswordMustMatch
messageAuthCredentialsRequired
messageAuthUsernameTaken
messageAuthEmailTaken
messageAuthNoUserFound
messageAuthMustBeLoggedIn
messageAuthMustIncludeTokens
messageGroupNotFound
messageUserOperationNotFound

--Npc.json--
donateText1
donateText2
donationDesc
payWithCard
payNote
amazonInstructions
moreClass
tourWelcome
tourExp
tourDailies
tourCron
tourHP
tourHabits
tourStats
tourGP
tourAvatar
tourScrollDown
tourMuchMore
tourAvatar
tourAvatarProceed
tourToDosBrief
tourDailiesBrief
tourDailiesProceed
tourHabitsBrief
TourHabitsProceed
tourRewardsBrief
tourRewardsArmoire
tourRewardsProceed

--Overview.json--
webStep1Text
webStep2Text
webStep3Text

--Pets.json--
stableMountMasterProgress
mountAchievement
beastMountMasterName
triadBingoAchievement
hatchAPot
displayNow
displayLater
earnedCompanion
useSaddle
raisedPet
earnedSteed
rideNow
rideLater
petKeyName
petKeyPop
petKeyBegin
petKeyInfo
petKeyInfo2
petKeyInfo3
petKeyInfo4
petKeyPets
petKeyMounts
petKeyBoth
confirmPetKey
petKeyNeverMind
gemsEach
foodWikiText
foodWikiUrl

--Tasks.json--
lotOfToDos
deleteToDosExplanation
newHabit
newHabitBulk
habitsDesc
expandChecklist
extraNotes
direction/Actions
difficultyHelpTitle
difficultyHelpContent
newDaily
newDailyBulk
dailysDesc
streakCounter
dailyRepeatHelpContent
weeklyRepeatHelpContent
newTodoBulk
newRewardBulk
newReward
sureDeleteCompletedTodos
dailiesRestingInInn
habitHelp1
habitHelp2
habitHelp3
newbieGuild
dailyHelp1
dailyHelp2
dailyHelp3
dailyHelp4
dailyHelp5
toDoHelp1
toDoHelp2
toDoHelp3
toDoHelp4
rewardHelp1
rewardHelp2
rewardHelp3
rewardHelp4
clickForHelp
strengthExample
intelligenceExample
perceptionExample
constitutionExample
counterPeriod
counterPeriodDay
counterPeriodWeek
counterPeriodMonth
repeatTypeHelpTitle
repeatType
repeatTypeHelp
repeatZero
monthlyRepeatHelpContent
yearlyRepeatHelpContent
summaryStart
yesterDailiesTitle
yesterDailiesOptionTitle
yesterDailiesDescription
repeatDayError
searchTasks
repeatDayError

--Settings.json--
showHeaderPop
restartTour
classTourPop
passwordSuccess
usernameSuccess
emailSuccess
usernameOrEmail
recurringText

--Subscriber.json--
subDescription
cancelSubInfoGoogle
cancelSubInfoApple
adminSub
morePlans
organizationSub
organizationSubText
hostingType
hostingTypeText
customDomain
customDomainText
maxPlayers
maxPlayersText
priSupport
priSupportText
timeSupport
timeSupportText
gameFeatures
gold2Gem
gold2GemText
infiniteGem
infiniteGemText
notYetPlan
buyGemsGoldTitle
becomeSubscriber
freeGemsTitle
buyGemsAllow1
buyGemsAllow2
purchaseGemsSeparately
subFreeGemsHow
seeSubscriptionDetails
mysticHourglassPopover
mysterySet201402
mysterySet201403
mysterySet201404
mysterySet201405
mysterySet201406
mysterySet201407
mysterySet201408
mysterySet201409
mysterySet201410
mysterySet201411
mysterySet201412
mysterySet201501
mysterySet201502
mysterySet201503
mysterySet201504
mysterySet201505
mysterySet201506
mysterySet201507
mysterySet201508
mysterySet201509
mysterySet201510
mysterySet201511
mysterySet201512
mysterySet201601
mysterySet201602
mysterySet201603
mysterySet201604
mysterySet201605
mysterySet201606
mysterySet201607
mysterySet201608
mysterySet201609
mysterySet201610
mysterySet201611
mysterySet201612
mysterySet201701
mysterySet201702
mysterySet201703
mysterySet201704
mysterySet201705
mysterySet201706
mysterySet201707
mysterySet201708
mysterySet201709
mysterySet201710
mysterySet201711
mysterySet301404
mysterySet301405
mysterySet301703
mysterySet301704
mysterySetwondercon
hourglassBuyItemConfirm
petsNotAllowedHourglass
mountsNotAllowedHourglass

--Rebirth.json--
rebirthBegin
rebirthStartOver
rebirthAdvList1
rebirthAdvList2
rebirthAdvList3
rebirthAdvList4
rebirthInherit
rebirthInList1
rebirthInList2
rebirthInList3
rebirthInList4
rebirthEarnAchievement
beReborn
confirmReborn

--Quests.json--
whereAreMyQuests
yourQuests
questsForSale
petQuests
unlockableQuests
goldQuests
questOwnerReceived
youWillReceive
questOwnerWillReceive
dropQuestCongrats
questSend
questSendBroken
questLater
questStartBroken
itemsToCollect
bossDmg1
bossDmg1Broken
bossDmg2
bossDmg2Broken
tavernBossInfo
tavernBossInfoBroken
bossColl1
bossColl1Broken
bossColl2
bossColl2Broken
questOwnerNotInPendingQuest
questOwnerNotInPendingQuestParty
questOwnerNotInRunningQuest
questOwnerNotInRunningQuestParty
questParticipants
noScrolls
scrollsText1
scrollsText2
scrollsPre
alreadyEarnedQuestLevel
alreadyEarnedQuestReward
completedQuests
mustComplete
mustLevel
mustInviteFriend
questWarning
questWarningBroken
whichQuestStart
getMoreQuests
unlockedAQuest
leveledUpReceivedQuest

--Noscript.json-- 
jsDisabledText

--Merch.Json--
merchandiseDescription
merch-teespring-summary
merch-teespring-goto
merch-teespring-mug-summary
merch-teespring-mug-goto
merch-teespring-eu-summary
merch-teespring-eu-gotof
merch-teespring-mug-eu-summary
merch-teespring-mug-eu-goto
merch-stickermule-summary
merch-stickermule-goto

--LoadingScreenTips.json--
tipTitle

--Limited.json--
limitedEdition
seasonalEdition
winterColors
gildedTurkey
polarBearPup
nye0
nye1
nye2
nye3
nye4
dateEndApril
dateEndMay
dateEndJune
dateEndJuly
dateEndAugust
dateEndOctober
dateEndNovember

### CODE REVIEW

File: website/common/locales/en/tasks.json [hunk 1]
```diff
@@ -160,26 +115,13 @@
...
   "taskRequiresApproval": "This task must be approved before you can complete it. Approval has already been requested",
   "taskApprovalHasBeenRequested": "Approval has been requested",
-  "taskApprovalWasNotRequested": "Only a task waiting for approval can be marked as needing more work",
   "approvals": "Approvals",
-  "approvalRequired": "Needs Approval",
-  "repeatZero": "Daily is never due",
-  "repeatType": "Repeat Type",
-  "repeatTypeHelpTitle": "What kind of repeat is this?",
-  "repeatTypeHelp": "Select \"Daily\" if you want this task to repeat every day or every third day, etc. Select \"Weekly\"if you want it to repeat on certain days of the week. If you select \"Monthly\" or \"Yearly\", adjust the Start Date to control which day of the month or year the task will be due on.",
+  "approvalRequired": "Approval Required",
```
Comment (paglias) [inline website/common/locales/en/tasks.json:124]: why some of these strings have been changed? There are other examples as well

**Review by paglias — COMMENTED** [review]
Thanks! I've noticed the content of some strings has been changed, can you revert the changes?

Also can you explain the process you used to find unused strings?

File: website/common/locales/en/tasks.json (hunk 1 above)
Comment (ghost) [reply website/common/locales/en/tasks.json:124]: Hey, the only thing I can think of as to why these would be saying changed was there were some strings I deleted accidentally and then re created them using reference from the original branch, but did it manually as I was typing opposed to reverting a change back using GIT. 

Aside from that I only removed strings as I found them unused.

Also I couldn't find an easier way other than using Visual Studio and performing searches on specific strings in the entire project to see where strings were being called and only removing those where they were never referenced outside of the locales.

Hope that helps

Comment (paglias) [reply website/common/locales/en/tasks.json:124]: Some strings have definitely different wording, see https://github.com/HabitRPG/habitica/pull/10369/files#diff-086914283aad8d25a218443de2119265L9

Comment (ghost) [reply website/common/locales/en/tasks.json:124]: Hmm ok. I think this is related to something I did to fix a previous issue.

I made a previous clone that was a bit older when I was working on it. I tried pushing my work up to my clone repo but it kept putting my work into a sub directory so I created a new clone of the Habitica rep recently and copied the locales folder into it. Perhaps there were some string changes made that I didn't account for between my old and new clone.

Definitely unintentional, so I'll revert my changes and see what I need to do to fix it.

Cheers

File: website/common/locales/en/contrib.json [hunk 2]
```diff
@@ -17,28 +17,20 @@
   "eliteThird": "When your <strong>third</strong> set of submissions is deployed, the <strong>Crystal Helmet</strong> will be available for purchase in the Rewards shop. As a bounty for your continued work, you will also receive <strong>3 Gems</strong>.",
   "eliteFourth": "When your <strong>fourth</strong> set of submissions is deployed, the <strong>Crystal Sword</strong> will be available for purchase in the Rewards shop. As a bounty for your continued work, you will also receive <strong>4 Gems</strong>.",
   "champion": "Champion",
-  "championFifth": "When your <strong>fifth</strong> set of submissions is deployed, the <strong>Crystal Shield</strong> will be available for purchase in the Rewards shop. As a bounty for your continued work, you will also receive <strong>4 Gems</strong>.",
```
Comment (Alys) [inline website/common/locales/en/contrib.json:20]: I think friendFirst, friendSecond, eliteThird, and eliteFourth can also be removed.

**Comment by negue** [comment]
@Inshabel could you also check the tests ? seems that some of those strings are still used

**Pull request closed without merging**